### PR TITLE
Event location

### PR DESCRIPTION
**Pull request by RobinNagpal**

### CODE REVIEW

File: frontend/src/components/utils/googlemap/marker/mapmarkercss.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  width: K_WIDTH,
+  height: K_HEIGHT,
+  left: -K_WIDTH / 2,
+  top: -K_HEIGHT / 2,
+
+  border: '5px solid #f44336',
+  borderRadius: K_HEIGHT,
+  backgroundColor: 'white',
+  textAlign: 'center',
+  color: '#3f51b5',
```
Comment (RobinNagpal) [inline frontend/src/components/utils/googlemap/marker/mapmarkercss.js]: color and font size can also be a variable

File: frontend/src/components/events/location/location.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  }
+
+  render() {
+    const location = this.props.location;
+    //var initialCenter = {lat: 45.304772, lng: -75.891355};
+    var initialCenter = { lat: parseFloat(location.latitude),lng: parseFloat(location.longitude) };
+    return (
+      <div className="row">
+        <div className="col-lg-4">
+            <div style={{width: '100%', height: '300px'}}>
```
Comment (RobinNagpal) [inline frontend/src/components/events/location/location.js]: we should not hardcode styles in HTML. You should move it to scss

File: frontend/src/components/events/location/directions/directions.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+class Directions extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    const directions = this.props.directions;
+    return (
+      <div className="">
```
Comment (RobinNagpal) [inline frontend/src/components/events/location/directions/directions.js]: do we need className here with empty value ?

File: frontend/src/components/events/location/address/address.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    const address = this.props.address;
+    return (
+      <div className="">
+        <div className="">{address.address1}</div>
+        <div className="">{address.address2}</div>
```
Comment (RobinNagpal) [inline frontend/src/components/events/location/address/address.js]: className ?

File: backend/src/database/1-seed.js [hunk 5]
```diff
@@ -28,17 +28,63 @@ module.exports.up = function (done) {
...
+        "latitude" : "45.304772",
+        "longitude" : "-75.891355",
+        "address" : {
+            "address1" : "hazeldean",
+            "address2" : "hazeldean2",
+            "city" : "kanata",
+            "state" : "ON",
+            "country" : "CA",
+            "postalCode" : "K2L2A2",
+            "unitNo" : "4"
```
Comment (debibains2003) [inline backend/src/database/1-seed.js]: Removed

File: backend/src/main/java/com/outdoor/model/Address.java [hunk 6]
```diff
@@ -0,0 +1,74 @@
+package com.outdoor.model;
+
+public class Address {
+private String address1;
```
Comment (debibains2003) [inline backend/src/main/java/com/outdoor/model/Address.java]: Done

File: frontend/src/components/utils/googlemap/marker/mapmarkercss.js (hunk 1 above)
Comment (debibains2003) [inline frontend/src/components/utils/googlemap/marker/mapmarkercss.js]: done

File: frontend/src/components/events/location/location.js (hunk 2 above)
Comment (debibains2003) [inline frontend/src/components/events/location/location.js]: removed

File: frontend/src/components/events/location/directions/directions.js (hunk 3 above)
Comment (debibains2003) [inline frontend/src/components/events/location/directions/directions.js]: removed

File: frontend/src/components/events/location/address/address.js (hunk 4 above)
Comment (debibains2003) [inline frontend/src/components/events/location/address/address.js]: removed

**Pull request merged**